include/wx/generic/private/grid.h

```diff
@@ -1227,6 +1236,11 @@ wxGetContentRect(wxSize contentSize,
                  int hAlign,
                  int vAlign);
 
+inline bool operator<(const wxPoint& pt1, const wxPoint& pt2)
```

Comment:
I'd prefer to define a custom comparator and pass it to `std::{set,map}` explicitly. Comparing points lexicographically doesn't always make sense and while it's unlikely to affect anything right now, when this is declared in a private header, it still might and it could be very annoying to run into problems due to this later.

Reply:
Done